functions/data_transformations.R

```diff
@@ -0,0 +1,65 @@
+library(purrr)
+library(tidyverse)
+library(httr)
+library(jsonlite)
+library(ggplot2)
+library(DescTools)
+library(magrittr)
+library(rlang)
+library(lubridate)
+library(anytime)
+
+# Problem 2
+
+transform_metadata_to_df <- function() {
```

Comment:
You need to have an input variable in this function. When you call the function using `transform_metadata_to_df(.)`, the function will not take any input. If you changed the first lines to this:
```
transform_metadata_to_df <- function(df) {
  df[[1]] %>% 
  ....
```

It would work fine. The rest of the function looks good.